pkg/oc/cli/secrets/subcommand.go

```diff
@@ -44,11 +44,6 @@ func NewCmdSecrets(name, fullName string, f *clientcmd.Factory, reader io.Reader
 		Run:     cmdutil.DefaultSubCommandRun(errOut),
 	}
 
-	newSecretFullName := fullName + " " + NewSecretRecommendedCommandName
-	cmds.AddCommand(NewCmdCreateSecret(NewSecretRecommendedCommandName, newSecretFullName, f, out))
```

Comment:
I expect a deprecation warning, but I still expect to be able to run `oc secret create`.  Is this not doing what I think it's doing?

Reply:
I believe they are still being added to the `oc secrets` tree here https://github.com/openshift/origin/pull/18093/files#diff-98f0c62da95151f788ac0edc23423802R200, but with the deprecation warning

Reply:
Yes, you still can run those, it'll just scream after you with the deprecation.